test/utils/bytecode.hpp

```diff
@@ -312,6 +312,16 @@ inline bytecode calldatacopy(bytecode src, bytecode dst, bytecode size)
     return std::move(size) + std::move(dst) + std::move(src) + OP_CALLDATACOPY;
 }
 
+inline bytecode returndatasize()
+{
+    return OP_RETURNDATASIZE;
+}
+
+inline bytecode returndatacopy(bytecode src, bytecode dst, bytecode size)
+{
+    return std::move(size) + std::move(dst) + std::move(src) + OP_RETURNDATACOPY;
```

Comment:
it's not the right order
```suggestion
    return std::move(size) + std::move(src) + std::move(dst) + OP_RETURNDATACOPY;
```

In tests you pass dst as first argument, so it works.

Similar problem with `calldatacopy(_` above.

Reply:
Ah, ok. I don't want to swap order artificially so I will just rename arguments.